libs/openchallenges/challenge/src/lib/challenge-contributors/challenge-contributors.component.ts

```diff
@@ -61,6 +62,10 @@ export class ChallengeContributorsComponent implements OnInit {
       );
   }
 
+  private sortOrgs(orgs: Organization[]): Organization[] {
+    return orgs.sort((a, b) => (a.name || '').localeCompare(b.name || ''));
```

Comment:
@rrchai I don't understand this. Are you doing a local sorting instead of relying on the backend sorting?

Reply:
I don't think the backend has supported to get a list of sorted contributions?

The single org request is used here after we got a list of org ids from `ChallenegContribution` service.

Reply:
I understand now. This note should be added to the first post of this PR to provide some context.